lua/nvim_lsp/jdtls.lua

```diff
@@ -0,0 +1,30 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+local cwd = vim.loop.cwd()
+
+local server_name = "jdtls"
+local bin_name = "jdtls"
+
+skeleton[server_name] = {
+  default_config = {
+    cmd = {"jdtls"};
```

Comment:
might as well use `bin_name` here if you define it above, or remove `local bin_name` above.